openstack/content-repo/bin/sleep-for

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -euo pipefail
+
+sleep_for=$1
+shift
+command="$@"
+
+# Run forever
+# Execute the provided command
+# Exit in case the command fails
+# Sleep X seconds, provided as first param
+while :; do
+  eval "$command" || exit 1
```

Comment:
Can be simplified to just `"$@" || exit 1`. `$@` cannot change over the course of the loop.